Libraries/LibWeb/CSS/ComputedValues.h

```diff
@@ -345,24 +345,6 @@ struct BorderRadiusData {
     CSS::LengthPercentage vertical_radius { InitialValues::border_radius() };
 };
 
-// FIXME: Find a better place for this helper.
-inline Gfx::ScalingMode to_gfx_scaling_mode(CSS::ImageRendering css_value, Gfx::IntRect source, Gfx::IntRect target)
```

Comment:
Moving this isn't part of the EXIF change, so it'd be good to put it in its own commit.